emscripten-version.txt

```diff
@@ -1 +1 @@
-"1.38.21"
+"1.38.22"
```

Comment:
Instead of bumping minor version, we should bump to `1.39.0`, because this is a breaking ABI change. (like we did in https://github.com/kripken/emscripten/pull/5916#issuecomment-355274893)

Reply:
I'm not opposed to that, but I don't think we have a strict rule here - even minor version number updates can break ABI, e.g. with an LLVM update. I did agree in the linked issue that we should do a major one because it was a large change, and because it could be particularly confusing and error-prone, but in this PR here the change is minor (unless I'm missing something), so I don't feel strongly either way.

Reply:
+1, I think this is worth bumping to 1.39
I don't recall whether we came to a strict rule but I hope we can eventually (soon) get to the point where we can have a strict rule of bumping the minor for ABI breaks.

Reply:
Sounds good. Then we should land this with a minor version, wait to see we are reasonably stable, then bump the major version, as documented here: https://github.com/kripken/emscripten/blob/incoming/docs/process.md#major-version-update-1xy-to-1x10